packages/combinator/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Dialtone vue component combinator",
   "scripts": {
     "lint": "eslint --ext .js,.vue --ignore-path .gitignore --fix src",
-    "test": "vue-cli-service test:unit --timeout 100000 --colors --recursive --glob *.test.js src"
+    "test:old": "vue-cli-service test:unit --timeout 100000 --colors --recursive --glob *.test.js src"
```

Comment:
Why are we doing this? combinator tests were messing up with the other tests?

Reply:
yeah if you just run `nx test` it runs all "test" scripts including this one, and they are broken because they haven't been migrated yet.